### PR TITLE
MF-324 - Added new database table for holding provider permissions

### PR DESCRIPTION
**Pull request by robinoconnor**

### CODE REVIEW

File: src/SFA.DAS.Reservations.Database/SFA.DAS.Reservations.Database.sqlproj [hunk 1]
```diff
@@ -70,5 +70,9 @@
     <Build Include="Tables\OutboxData.sql" />
     <Build Include="Tables\ClientOutboxData.sql" />
     <Build Include="Tables\UserRuleNotification.sql" />
+    <Build Include="Tables\ProviderPermission.sql" />
+  </ItemGroup>
+  <ItemGroup>
+    <RefactorLog Include="SFA.DAS.Reservations.Database.refactorlog" />
```
Comment (dashton82) [inline src/SFA.DAS.Reservations.Database/SFA.DAS.Reservations.Database.sqlproj]: Can be removed

File: src/SFA.DAS.Reservations.Database/SFA.DAS.Reservations.Database.refactorlog [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Operations Version="1.0" xmlns="http://schemas.microsoft.com/sqlserver/dac/Serialization/2012/02">
+  <Operation Name="Rename Refactor" Key="040161ab-6dc1-4475-8478-8c962cae99c3" ChangeDateTime="10/10/2019 11:48:37">
+    <Property Name="ElementName" Value="[dbo].[ProviderPermission].[Id]" />
+    <Property Name="ElementType" Value="SqlSimpleColumn" />
+    <Property Name="ParentElementName" Value="[dbo].[ProviderPermission]" />
+    <Property Name="ParentElementType" Value="SqlTable" />
+    <Property Name="NewName" Value="AccountId" />
+  </Operation>
+</Operations>
```
[the remark this answers is not in the document]
Comment (robinoconnor) [reply src/SFA.DAS.Reservations.Database/SFA.DAS.Reservations.Database.refactorlog]: done

File: src/SFA.DAS.Reservations.Database/SFA.DAS.Reservations.Database.sqlproj (hunk 1 above)
Comment (robinoconnor) [reply src/SFA.DAS.Reservations.Database/SFA.DAS.Reservations.Database.sqlproj]: done

File: src/SFA.DAS.Reservations.Database/Tables/ProviderPermission.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[ProviderPermission]
+(
+	[AccountId] BIGINT NOT NULL , 
+    [AccountLegalEntityId] BIGINT NOT NULL, 
+    [Ukprn] BIGINT NOT NULL, 
+    [CanCreateCohort] BIT NOT NULL, 
+    PRIMARY KEY ([AccountId],[AccountLegalEntityId],[Ukprn])
+)
```
[the remark this answers is not in the document]
Comment (robinoconnor) [reply src/SFA.DAS.Reservations.Database/Tables/ProviderPermission.sql]: done

**Pull request merged**